services/repository/repository.go

```diff
@@ -51,6 +52,10 @@ func DeleteRepository(ctx context.Context, doer *user_model.User, repo *repo_mod
 		return err
 	}
 
+	if err := secret_service.DeleteSecretsByRepoID(ctx, repo.ID); err != nil {
```

Comment:
I thought you would simply use `models.DeleteRepository#130-152` for it…

Reply:
Solved in the new PR.